src/main/java/org/openjdk/jextract/clang/libclang/CXString.java

```diff
@@ -47,13 +47,13 @@ public class CXString {
         return CXString.data$VH;
     }
     public static MemoryAddress data$get(MemorySegment seg) {
-        return (jdk.incubator.foreign.MemoryAddress)CXString.data$VH.get(seg);
+        return (java.lang.foreign.MemoryAddress)CXString.data$VH.get(seg);
     }
     public static void data$set( MemorySegment seg, MemoryAddress x) {
         CXString.data$VH.set(seg, x);
     }
     public static MemoryAddress data$get(MemorySegment seg, long index) {
-        return (jdk.incubator.foreign.MemoryAddress)CXString.data$VH.get(seg.asSlice(index*sizeof()));
+        return (java.lang.foreign.MemoryAddress)CXString.data$VH.get(seg.asSlice(index*sizeof()));
```

Comment:
Maybe jextract should not emit a fully qualified name for the cast target? There's an import anyways.

Reply:
Yes, pre-existing issue I think. Will address it in another bug